app/helpers/application_helper.rb

```diff
@@ -8,6 +8,14 @@ def app_name
     #NUCore.app_name
   end
 
+  def app_name_link_text
+    if current_ability.can?(:act_as, current_facility)
```

Comment:
If you're an admin and are in the front-end (e.g. placing an order), both ordering on behalf of and as yourself, you'll see "NU Core Admin". I'm thinking this might not be the correct logic. Although, it becomes moot if we're able to drop the logic.